.github/workflows/test.yaml

```diff
@@ -84,7 +84,6 @@ jobs:
       fail-fast: false      
       matrix:
         gochannel:
-          - 1.13
```

Comment:
Please just merge/rebase to master and remove this commit that changes this. This is fixed now.